extmod/modframebuf.c

```diff
@@ -491,7 +497,7 @@ STATIC mp_obj_t framebuf_blit(size_t n_args, const mp_obj_t *args) {
         (y >= self->height) ||
         (-x >= source->width) ||
         (-y >= source->height)
-    ) {
+        ) {
         // Out of bounds, no-op.
         return mp_const_none;
     }
```

Comment:
Readability before was better.

Reply:
there might be some uncrustify options that improve this, but for now I think we just leave these few cases as-is, maybe fix them later on